src/silo/config/database_config.cpp

```diff
@@ -144,7 +139,12 @@ struct convert<silo::config::DatabaseMetadata> {
       if (node["generateIndex"].IsDefined()) {
          metadata.generate_index = node["generateIndex"].as<bool>();
       } else {
-         metadata.generate_index = metadata.type == silo::config::ValueType::PANGOLINEAGE;
+         metadata.generate_index = false;
+      }
+      if (node["lineageIndex"].IsDefined()) {
+         metadata.lineage_index = node["lineageIndex"].as<bool>();
```

Comment:
As discussed: Let's rename it to `generateLineageIndex`
```suggestion
      if (node["generateLineageIndex"].IsDefined()) {
         metadata.generateLineageIndex = node["generateLineageIndex"].as<bool>();
```